charts/chainlink-cluster/dashboard/cmd/deploy.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"github.com/K-Phoen/grabana/dashboard"
+	lib "github.com/smartcontractkit/chainlink/dashboard-lib/lib"
+	core_don "github.com/smartcontractkit/chainlink/dashboard-lib/lib/core-don"
+	k8spods "github.com/smartcontractkit/chainlink/dashboard-lib/lib/k8s-pods"
+	waspdb "github.com/smartcontractkit/wasp/dashboard"
+)
+
+const (
+	DashboardName = "Chainlink Cluster (DON)"
+)
+
+func main() {
+	cfg := lib.ReadEnvDeployOpts()
+	db := lib.NewDashboard(DashboardName, cfg,
+		[]dashboard.Option{
+			dashboard.AutoRefresh("10s"),
+			dashboard.Tags([]string{"experimental", "generated"}),
+		},
+	)
+	db.Add(
+		core_don.New(
+			core_don.Props{
+				PrometheusDataSource: cfg.DataSources.Prometheus,
+				PlatformOpts:         core_don.PlatformPanelOpts(cfg.Platform),
+			},
+		),
+	)
+	db.Add(
+		k8spods.New(
+			k8spods.Props{
+				PrometheusDataSource: cfg.DataSources.Prometheus,
+				LokiDataSource:       cfg.DataSources.Loki,
+			},
+		),
+	)
```

Comment:
Could we add condition to only add it for platform = kubernetes

Reply:
Good catch, will do